pages/desktop/details.py

```diff
@@ -69,11 +69,6 @@ class Details(Base):
     _reviews_locator = (By.CSS_SELECTOR, "section#reviews div")
     _add_review_link_locator = (By.ID, "add-review")
 
-    _add_to_collection_locator = (By.CSS_SELECTOR, ".collection-add.widget.collection")
-    _add_to_collection_widget_button_locator = (By.CSS_SELECTOR, ".collection-add-login .register-button .button")
-    _add_to_collection_widget_login_link_locator = (By.CSS_SELECTOR, "div.collection-add-login p:nth-child(3) > a")
-    _add_to_favorites_widget_locator = (By.CSS_SELECTOR, 'div.widgets > a.favorite')
```

Comment:
This is referenced in `click_add_to_favorites` and `is_addon_marked_as_favorite`. These are not used by the test `test_that_add_to_collection_flyout_for_anonymous_users` so removal of the locator will cause failures when calling these methods. Incidentally, these methods are referenced in `test_user_my_favorites_page` within `test_collections.py` which is currently failing due to element not found, so we may just need to remove that test too.